Added doc about rand-seed and rand-address (#2450)
iotaledger/goshimmer: documentation/docs/tooling/rand_seed_and_rand_address.md

---

> description: Running these tools let you generate random seeds and addresses through a simple command.

Reviewer: ```suggestion
description: 'You can use the rand-address and rand-seed tools to generate random seeds and addresses through a simple command.'
```

keywords:
- address
- seed
- public key
- private key
- generate
- generation
---

> # Rand seed and rand address

Reviewer: ```suggestion
# Rand Seed and Rand Address

You can use the [`rand-address`](#rand-address) and [`rand-seed`](#rand-seed) tools to generate addresses and seeds in a single command.
```

> # Rand seed and rand address

Reviewer: ```suggestion
# Rand Seed and Rand Address

You can use the [`rand-address`](#rand-address) and [`rand-seed`](#rand-seed) tools to generate addresses and seeds in a single command. 

You can later use the generated data to, for example, set up the exact same nodes in a private network.
```

## Rand address

> Rand address is a tool realized through a Go script that let you generate a random address. Its execution is pretty simple:

Reviewer: ```suggestion
## Rand Address

You can use the `rand-address` tool to generate a random address by running the following command from the `tools/rand-address` directory:
```

> Rand address is a tool realized through a Go script that let you generate a random address. Its execution is pretty simple:

Reviewer: ```suggestion
You can use the `rand-address` tool to generate a random address by running the following command from the `tools/rand-address` directory:
```

> ```shell
> cd tools/rand-address
> go run main.go
> 13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
> ```
> 
> Right after the execution, you will have a new randomly generated address.

Reviewer: ````suggestion
```shell
cd tools/rand-address
go run main.go
```

### Expected Output

The script will output a Base58 string representing the newly generated address, for example:

```shell
13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
```
````

> ## Rand seed

Reviewer: ```suggestion
## Rand Seed
```

> The rand seed tool is a CLI script a bit more complete than the previous one. It let you generate in a text file a seed (encoded in base58 and base64), its relative identity and its public key. The execution is still simple:

Reviewer: ```suggestion
## Rand Seed

You can use the `rand-address` tool to generate a text file with the following:

* A [seed](/../tutorials/send_transaction#seed), represented in Base64 and Base58.
* The seed's relative identity, as a Base58 string. 
* The relative identity's public, key in Base58.
```

```shell
cd tools/rand-seed
go run main.go
```

> The script will generate a `.txt` file containing, for example, the following:
> 
> ```shell
> base64:ri9C8oAT3IPsus2j+IllMbW2B3nOqe4uC56zfr344zY=
> base58:CiwjnjMRwEbCGiATWjNsrVptBTNH13AHrVNmG31KK9cy
> Identity - base58:BCUnRc6c
> Identity - base58:BCUnRc6cv4YVnB3Rw5DDfdsFuVVUW97MyLEBzWxHqfQj
> Public Key - base58:Ht9VR8qAgmruDPzsQbak3AJvXcJY6q6Mxyaz4pDicDEw
> ```
> 
> These tools are useful if you have to run, for example, a Docker Private Network and you want to setup the exact nodes running in the network.

Reviewer: ````suggestion
### Expected Output

The script will generate a `random-seed.txt` file in the current working directory, for example:

```plaintext
base64:ri9C8oAT3IPsus2j+IllMbW2B3nOqe4uC56zfr344zY=
base58:CiwjnjMRwEbCGiATWjNsrVptBTNH13AHrVNmG31KK9cy
Identity - base58:BCUnRc6c
Identity - base58:BCUnRc6cv4YVnB3Rw5DDfdsFuVVUW97MyLEBzWxHqfQj
Public Key - base58:Ht9VR8qAgmruDPzsQbak3AJvXcJY6q6Mxyaz4pDicDEw
```
````

---
description: 'You can use the rand-address and rand-seed tools to generate random seeds and addresses through a simple command.'
keywords:
- address
- seed
- public key
- private key
- generate
- generation
---
# Rand Seed and Rand Address

You can use the [`rand-address`](#rand-address) and [`rand-seed`](#rand-seed) tools to generate addresses and seeds in a single command.

## Rand Address

You can use the `rand-address` tool to generate a random address by running the following command from the `tools/rand-address` directory:

```shell
cd tools/rand-address
go run main.go
```

### Expected Output

The script will output a Base58 string representing the newly generated address, for example:

```shell
13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
```

## Rand Seed

You can use the `rand-address` tool to generate a text file with the following:

> * A [seed](/../tutorials/send_transaction#seed), represented in Base64 and Base58.

Reviewer 2: ```suggestion
* A [seed](../tutorials/send_transaction#seed), represented in Base64 and Base58.
```

Reviewer 2: Link needs to point to relative instead of absolute path.

---
description: 'You can use the rand-address and rand-seed tools to generate random seeds and addresses through a simple command.'
keywords:
- address
- seed
- public key
- private key
- generate
- generation
---
# Rand Seed and Rand Address

You can use the [`rand-address`](#rand-address) and [`rand-seed`](#rand-seed) tools to generate addresses and seeds in a single command.

## Rand Address

You can use the `rand-address` tool to generate a random address by running the following command from the `tools/rand-address` directory:

```shell
cd tools/rand-address
go run main.go
```

### Expected Output

The script will output a Base58 string representing the newly generated address, for example:

```shell
13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
```

> 
> ## Rand Seed

Reviewer: ```suggestion
```

Sorry, it seems Ieft a double header here while suggesting.
